src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -877,6 +877,7 @@ public void parse(InputFile inputFile, Format ext, int type, boolean thumbOnly,
 					// Create the thumbnail image using the Thumbnailator library
 					try {
 						thumb = ImagesUtil.scaleImage(Files.readAllBytes(file.toPath()), 320, 320, ScaleType.MAX, ImageFormat.JPEG, false);
+						ffmpeg_parsing = false;
```

Comment:
@valib That was my problem as well. I was trying to figure out what is safe to do, but the structure of the code is so bad that it's very hard to have an overview over all possible paths. 

It might be as simple as ```if (!thumbOnly || (isVideo() && !configuration.isUseMplayerForVideoThumbs()))```, but I'm just not sure if that will cover all situations.